dev/com.ibm.ws.concurrent.persistent_fat_multiple/build.gradle

```diff
@@ -8,6 +8,71 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
+ 
+def isLocalBuild = Boolean.valueOf(System.getProperty("fat.test.localrun"))
+def dbType = System.properties.getProperty("fat.bucket.db.type")
 
+configurations {
+  anonymousJDBC
+}
 
-addRequiredLibraries.dependsOn addDerby
+dependencies {
+  //Test container libraries
+  requiredLibs 'org.testcontainers:testcontainers:1.12.0',
+               'org.testcontainers:database-commons:1.12.0',
+               'org.testcontainers:jdbc:1.12.0',
+               'org.testcontainers:oracle-xe:1.12.0',
+               'org.testcontainers:mssqlserver:1.12.0',
+               'org.testcontainers:postgresql:1.12.0',
+               'org.testcontainers:mockserver:1.12.0',
+               'org.testcontainers:db2:1.12.0',
+               'org.apache.commons:commons-compress:1.18',
+               'org.rnorth.duct-tape:duct-tape:1.0.7',
+               'org.rnorth.visible-assertions:visible-assertions:2.1.2',
+               'org.rnorth:tcp-unix-socket-proxy:1.0.2',
+               'net.java.dev.jna:jna:5.2.0',
+               'org.slf4j:slf4j-api:1.7.7',
+               'org.slf4j:slf4j-jdk14:1.7.7'
+
+  //JDBC driver - depends on database type
+  switch(dbType){
```

Comment:
at the time we build the FAT (i.e. the time this code runs) we won't know what DB will be used yet. Furthermore, the same autoFVT.zip produced by this build may be re-used to run against every DB.
The easiest way around this would be to just package every (available) JDBC all the time into the autoFVT.zip, since we don't know which JDBC driver will be used at runtime.